pkg/server/plugin/datastore/sql/sql_test.go

```diff
@@ -60,15 +61,21 @@ func (s *PluginSuite) SetupSuite() {
 
 	s.cacert, _, err = testutil.LoadCAFixture()
 	s.Require().NoError(err)
+
 }
 
 func (s *PluginSuite) SetupTest() {
 	s.dir = s.TempDir()
 	s.ds = s.newPlugin()
 }
 
+func (s *PluginSuite) AfterTest(_, testName string) {
```

Comment:
nit: TearDownTest is the usually companion method to SetupTest

Reply:
I stopped reading the docs when I got to TearDownSuite, my bad :)